content/docs/deployment/csminstallationwizard/src/static/js/generate-yaml.js

```diff
@@ -230,7 +240,7 @@ function createYamlString(yamlTpl, yamlTplValues, driverParam, CONSTANTS_PARAM)
 	yamlTpl = yamlTpl.replaceAll("$CERT_MANAGER_ENABLED", yamlTplValues.certManagerEnabled);
 	yamlTpl = yamlTpl.replaceAll("$OBSERVABILITY_CERT_MANAGER_ENABLED", !yamlTplValues.certManagerEnabled);
 	yamlTpl = yamlTpl.replaceAll('      - storageArrayId: ""', '#      - storageArrayId: ""');
-	yamlTpl = yamlTpl.replaceAll('        endpoint: ""', '#       endpoint: ""');	
+	yamlTpl = yamlTpl.replaceAll('        endpoint: ""', '#       endpoint: ""');
```

Comment:
Line number 243 and 244 are same. Duplicate line can be removed

Reply:
hi @niranjan-n1, this is related to a previous PR, https://github.com/dell/csm-docs/pull/804 for PowerMax replication.